string camera_mode_request
---
